assignments/callbacks.js

```diff
@@ -27,12 +27,22 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, function (length) {
+  console.log(length);
+});
+
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb(arr[arr.length - 1]);
 }
```

Comment:
You're close! don't manipulate the array here. Just pass the arr into the callback and do all the manipulation in the callback function. See above.